app/views/phrasing_phrases/index.html.haml

```diff
@@ -1,9 +1,12 @@
-#search
+#search{ style: "display:inline-block;" }
```

Comment:
We don't write inline css. if you need this, you should but it in a proper css file.